imagetest/cmd/manager/main.go

```diff
@@ -40,6 +40,7 @@ var (
 	validate        = flag.Bool("validate", false, "validate all the test workflows and exit")
 	outPath         = flag.String("out_path", "junit.xml", "junit xml path")
 	gcsPath         = flag.String("gcs_path", "", "GCS Path for Daisy working directory")
+	localPath       = flag.String("local_path", "", "path where test output files are stored, can be modified for local testing")
```

Comment:
Which docker paths would be broken by this? The goal of this variable is that with the default value of an empty string, the paths in main.go and testworkflow.go would remain the same as before

For example, the testpackage in testworkflow.go previously was
```
twf.wf.Sources["testpackage"] = fmt.Sprintf("/%s.%s.test", twf.Name, arch)
```
and it would become this, which should be the same string if localPath is unset
```
twf.wf.Sources["testpackage"] = fmt.Sprintf("%s/%s.%s.test", localPath, twf.Name, arch)
```

Reply:
Oh, you're right, I misread. Sorry!